wolox-books/src/app/components/auth/auth.component.html

```diff
@@ -0,0 +1,4 @@
+<nav class="container-nav">
```

Comment:
You could create a global component named navbar, in the future, you might have various screens using it :)

Reply:
create this componanete since in this card they are suggested to be called that and create it as a component for it to be used in other views if needed

Reply:
Okay!!